sessionctx/variable/session.go

```diff
@@ -1655,18 +1685,20 @@ func (s *SessionVars) GetParseParams() []parser.ParseParam {
 func (s *SessionVars) SetUserVar(varName string, svalue string, collation string) {
```

Comment:
How about renaming this function to `SetStringUserVar(name, strVal, collation string)`?